includes/models/model.llms.quiz.attempt.php

```diff
@@ -437,17 +439,39 @@ private function get_new_questions() {
 				}
 
 				$questions[] = array(
-					'id'      => $question->get( 'id' ),
-					'earned'  => 0,
-					'points'  => $question->supports( 'points' ) ? $question->get( 'points' ) : 0,
-					'answer'  => null,
-					'correct' => null,
+					'id'                      => $question->get( 'id' ),
+					'earned'                  => 0,
+					'points'                  => $question->supports( 'points' ) ? $question->get( 'points' ) : 0,
+					'answer'                  => null,
+					'correct'                 => null,
+					'include_bank_variations' => 'yes' === $question->get( 'include_bank_variations' ) ? true : false,
```

Comment:
Hey Akash, what is this about? why is it here and not in the add-on?

Reply:
My bad, I've moved this in the add-on and will try to move everything in the addon now.

This is a setting for making a question required to be included in the question bank.